CodeGenerator/Sources/CodeGenerator/main.swift

```diff
@@ -16,63 +32,78 @@ var errorShapeMap: [String: String] = [:]
 let fsLoader = FileSystemLoader(paths: [Path("\(rootPath())/CodeGenerator/Templates/")])
 let environment = Environment(loader: fsLoader)
 
-for index in 0..<apis.count {
-    let api = apis[index]
-    let doc = docs[index]
+for model in models {
 
     group.enter()
+
     DispatchQueue.global().async {
```

Comment:
we should `defer group.leave()` in here.

Reply:
Happy to do that. You’re looking at inherited code now